docs/translations/translation-basics.md

```diff
@@ -0,0 +1,59 @@
+# Translation basics
+
+## Localization functions
+
+Since [WordPress 2.1 "Ella"](https://wordpress.org/support/wordpress-version/version-2-1/), WordPress offers i18n for PHP files, and since [WordPress 5.0 "Bebo"](https://wordpress.org/support/wordpress-version/version-5-0/), it also offers i18n for JS files. Handling translations is pretty straight forward. Both PHP and JS handle translations similar. WordPress offers the functions:
+
+-   `__()` → Available in PHP & JS/TS.
+-   `_e()` → Available in PHP only.
+-   `_ex()` → Available in PHP only.
+-   `_n()` → Available in PHP & JS/TS.
+-   `_x()` → Available in PHP & JS/TS.
+-   `_nx()` → Available in PHP & JS/TS.
+-   `esc_html__()` → Available in PHP only.
+-   `esc_html_e()` → Available in PHP only.
+-   `esc_html_x()` → Available in PHP only.
+-   `esc_attr__()` → Available in PHP only.
+-   `esc_attr_e()` → Available in PHP only.
+-   `esc_attr_x()` → Available in PHP only.
```

Comment:
What do you think about linking to these functions on the WordPress code reference site?

Reply:
In a previous version, I actually linked these functions to their corresponding reference. I am not sure why I removed these links, but having them is handy. I just added them again.